.github/workflows/stale.yml

```diff
@@ -0,0 +1,25 @@
+name: "Close stale issues"
+
+on:
+  schedule:
+  - cron: "0 * * * *"
```

Comment:
Temporarily set to run every hour to test this PR. This will be set to run every day (`"0 0 * * *"`) prior to merging.

Reply:
Removed since this is being tested in another repo.